admin_manual/installation/system_requirements.rst

```diff
@@ -2,65 +2,97 @@
+          reliable scaling) and support, we strogly recommend you to check out our `enterprise support
+          <https://nextcloud.com/enterprise/>`_.
+
++------------------+-----------------------------------------------------------------------+
+| Platform         | Options                                                               |
++==================+=======================================================================+
+| Operating System | - **Ubuntu 16.04/18.04 LTS** (recommended)                            |
+|                  | - **Red Hat Enterprise Linux 7** (recommended)                        |
+|                  | - Debian 8 (Jessie), 9 (Strech)                                       |
+|                  | - SUSE Linux Enterprise Server 11 with SP3 & 12                       |
+|                  | - openSUSE Leap 42.1+                                                 |
+|                  | - CentOS 7                                                            |
++------------------+-----------------------------------------------------------------------+
+| Memory           | - **Apache 2.4 with ``mod_php``, ``php-fpm``** (recommended)          |
++------------------+-----------------------------------------------------------------------+
+| Database         | - **MySQL or MariaDB 5.5+** (recommended)                             |
+|                  | - Oracle Database 11g (*only as part of an enterprise subscription*)  |                                                
+|                  | - PostgreSQL 9/10                                                     |
+|                  | - SQLite (*only recommended for testing and minimal-instances*)       |                                                     
++------------------+-----------------------------------------------------------------------+
+| Web server       | - **Apache 2.4 with ``mod_php``, ``php-fpm``** (recommended)          |
+|                  | - nginx with ``php-fpm``                                              |
++------------------+-----------------------------------------------------------------------+
+| PHP Runtime      | - 7.0                                                                 |
+|                  | - 7.1                                                                 |
+|                  | - 7.2                                                                 |
++------------------+-----------------------------------------------------------------------+
+| *Hypervisors*    | - Hyper-V                                                             |
+|                  | - VMWare ESX                                                          |
+|                  | - Xen                                                                 |
+|                  | - KVM                                                                 |
```

Comment:
We never had issues with Hypervisors as they are at least two levels below our stack. Maybe we should drop this completely.